07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md

```diff
@@ -69,30 +71,48 @@ POST https://<WebAPI-URL>/adhocscan/subscription/{subscriptionId}/ControlScanRes
 }
 ```
 **Sample Response**
+> **Note:** The response object shown here might be shortened for readability.
+
 ``` JSON
-[
-    {
-        "ControlName": "Azure_Storage_DP_Encrypt_In_Transit",
-        "ControlDisplayName": "Enable Secure transfer to storage accounts",
-        "Status": "Failed",
-        "ResourceName": "htc-str-pre-prod-1",
-        "ResourceGroupName": "AzSKTestRG",
-        "ResourceID": "/subscriptions/48f73dab-a34c-49a5-b5f5-db1285d422c/resourceGroups/HTCRG/providers/Microsoft.Storage/storageAccounts/htc-str-pre-prod-1",
-        "IsBaseline": "true",
-        "Severity": "High",
-        "StatusReason": "Storage secure transfer is not enabled",
-        "SubscriptionId": "48f73dab-a34c-49a5-b5f5-db1285d422c",
-        "OrgTenantId": "524c134f-eae7-4d9e-b007-ebf8f9d6468f",
-        "ExceptionExpiryDate": "",
-        "Justification": "",
-        "Description": "HTTPS protocol must be used for accessing Storage Account resources",
-        "Category": "Encrypt data in transit",
-        "Remediation": "Run command 'Set-AzStorageAccount -ResourceGroupName <RGName> -Name <StorageAccountName> -EnableHttpsTrafficOnly `$true'. Run 'Get-Help Set-AzStorageAccount -full' for more help.",
-        "ExceptionWorkFlowStatus": ""
-    }
-]
+{
+    "data":
+    [
+        {
+            "ControlName": "Azure_Storage_DP_Encrypt_In_Transit",
+            "ControlDisplayName": "Enable Secure transfer to storage accounts",
+            "Status": "Failed",
+            "ResourceName": "htc-str-pre-prod-1",
+            "ResourceGroupName": "TestRG",
+            "ResourceID": "/subscriptions/48f73dab-a34c-49a5-b5f5-db1285d422c/resourceGroups/HTCRG/providers/Microsoft.Storage/storageAccounts/htc-str-pre-prod-1",
+            "IsBaseline": "true",
+            "Severity": "High",
+            "StatusReason": "Storage secure transfer is not enabled",
+            "SubscriptionId": "48f73dab-a34c-49a5-b5f5-db1285d422c",
+            "OrgTenantId": "524c134f-eae7-4d9e-b007-ebf8f9d6468f",
+            "ExceptionExpiryDate": "",
+            "Justification": "",
+            "Description": "HTTPS protocol must be used for accessing Storage Account resources",
+            "Category": "Encrypt data in transit",
+            "Remediation": "Run command 'Set-AzStorageAccount -ResourceGroupName <RGName> -Name <StorageAccountName> -EnableHttpsTrafficOnly `$true'. Run 'Get-Help Set-AzStorageAccount -full' for more help.",
+            "ExceptionWorkFlowStatus": ""
+        }
+    ],
+    "pageSize": 100,
+    "skipToken": "eyJQYWdlTnVtYmVyIjoyfQ==",
+    "totalPages": 5,
+    "totalRecords": 450,    
+}
 ```
 
+To get the next page of the results, send a request to the URL with skipToken. The skipToken contains next page number of control scan result. Continue sending requests with the skipToken until it no longer contains a skipToken in the returned results.
```

Comment:
We should mention somewhere skipToken will not be there if there are no more pages left,

Reply:
We have mentioned - "Continue sending requests with the skipToken until it no longer contains a skipToken in the returned results."

If this is not clear, we have added in response table as well.